migrator/src/main/java/com/redhat/lightblue/migrator/AbstractController.java

```diff
@@ -17,6 +18,8 @@
 public abstract class AbstractController extends Thread {
 
     private static final Logger LOGGER=LoggerFactory.getLogger(AbstractController.class);
+
+    private final String callerId=UUID.randomUUID().toString();
```

Comment:
Forgot that it does. Will remove.

On Wed, Sep 9, 2015 at 7:14 AM, Dennis Crissman <notifications@github.com>
wrote:

> In
> migrator/src/main/java/com/redhat/lightblue/migrator/AbstractController.java
> <https://github.com/lightblue-platform/lightblue-migrator/pull/248#discussion_r39039723>
> :
>
> > @@ -17,6 +18,8 @@
> >  public abstract class AbstractController extends Thread {
> >
> >      private static final Logger LOGGER=LoggerFactory.getLogger(AbstractController.class);
> > +
> > +    private final String callerId=UUID.randomUUID().toString();
>
> Locking will use a random UUID by default. No reason to duplicate that
> effort.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/lightblue-platform/lightblue-migrator/pull/248/files#r39039723>
> .
>